disco_aws_automation/disco_subnet.py

```diff
@@ -154,6 +155,15 @@ def create_peering_routes(self, peering_conn_id, cidr):
 
             self._route_table = self._find_route_table()
 
+    def delete_route(self, destination_cidr_block):
+        """ Delete the route to the destination CIDR block from the route table """
+        delete_params = {
+            'RouteTableId': self.route_table['RouteTableId'],
+            'DestinationCidrBlock': destination_cidr_block
+        }
+        self.boto3_ec2.delete_route(**delete_params)
+        self._route_table = self._find_route_table()
```

Comment:
After delete a route, we need to refresh the data in `_route_table` otherwise it would still have the route that's already deleted.